clutch/demo/alpha.lurk

```diff
@@ -121,7 +124,36 @@
 
 
 !(:commit (fn<-db ledger))
-0x36283c9973a837574e3e5a15815b60a541df265b0f218ab048e99a7c44ee3769
 
 !(:call 0x36283c9973a837574e3e5a15815b60a541df265b0f218ab048e99a7c44ee3769 '(1 0 2))
-;; !(:prove)
+!(:prove)
```

Comment:
Won't folks need the proof for this in order to not be stuck here for 2h?
For reference, it's at 
https://gist.github.com/huitseeker/f287daaa4fab64ee14d3d5b4fd7b06ec

Reply:
Yes, good point. However, as an aside — I tried changing the reduction-count to 10, and now I can generate all of these proofs fairly quickly and without errors. I'll update the PR, then we can merge. I no longer need a swarm of helpful provers for this purpose.

Reply:
You don't, but I would bet a certain @johnchandlerburnham would. His ambitions are much more size-challenged, or so I've heard.

Reply:
I was only referring to the call-to-provers this PR itself represented — not to the general value of proving services/volunteers.